packages/runtime/src/useCases/transport/messages/SendMessage.ts

```diff
@@ -58,6 +58,18 @@ export class SendMessageUseCase extends UseCase<SendMessageRequest, MessageDTO>
 
     private async validateMessageContent(content: any, recipients: string[]) {
         const transformedContent = Serializable.fromUnknown(content);
+        if (
+            !(
+                transformedContent instanceof Mail ||
+                transformedContent instanceof ResponseWrapper ||
+                transformedContent instanceof Notification ||
+                transformedContent instanceof ArbitraryMessageContent ||
+                transformedContent instanceof Request
+            )
+        ) {
+            return RuntimeErrors.general.invalidPropertyValue("The content type of a Message must be Mail, Request, ResponseWrapper, Notification or ArbitraryMessageContent.");
```

Comment:
I see some problems with the error message (this also will apply to other places)

- what is "the content type" and how does the user know what you mean by that
- Mail etc. should be in single ticks

I suggest
```suggestion
            return RuntimeErrors.general.invalidPropertyValue("The content's '@type‘ of a Message must be ‘Mail‘, ‘Request‘, ‘ResponseWrapper‘, ‘Notification‘ or ‘ArbitraryMessageContent‘.");
```
but I would be happy to hear other thoughts (cc @Milena-Czierlinski, @britsta, @slavistan)

Reply:
For me "content type" is clear, but I don't know how an external user would think about it and I don't mind your suggestion. Only note, please, that you use varying single ticks.

Reply:
true, please don't use these strange ticks @Magnus-Kuhn :D

Reply:
In principle, please do not use single ticks to enclose types from the data model overview. We don't do this otherwise either, but only for actual **values** (for example 'ID of a LocalAttribute'). Compare with the comment on this from @jkoenig134 in this PR https://github.com/nmshd/runtime/pull/179#issuecomment-2225292069.

Therefore, for reasons of consistency, `"The 'content' of a Message must contain either a Mail, a Request, a ResponseWrapper, a Notification or an ArbitraryMessageContent."` would be my suggestion.

Reply:
"The content must contain" was a chalk on chalkboard moment for me - but I like it with "The content must be"

Reply:
😂😂

Reply:
@britsta IMO this message is not clear enough.

I talk about actual strings, we use 's when we describe ids or other strings, so my formatting should be correct.

Reply:
So when I consider my comment from the other PR that you linked 

`"The content's @type of a Message must be 'Mail', 'Request', 'ResponseWrapper', 'Notification' or 'ArbitraryMessageContent'."`

Reply:
When not talking about properties I would write it as `"The content of a Message must either be a Mail, Request, ResponseWrapper, Notification or an ArbitraryMessageContent."`

Reply:
Since in this variant proposed by you the types are not used directly as words in the sentence, but are referred to abstractly, it is okay for me to enclose them here exceptionally with single ticks (even though I would not do it to keep consistency as much as possible). However, types from the data model overview should generally not be enclosed with single ticks in error messages and the documentation as well.

Reply:
> Since in this variant proposed by you the types are not used directly as words in the sentence

again, in my first variant I **don't** talk about types, I talk about the value that can be stored in the `@type` field of the content. This is very convenient for the user as he don't has to understand our wording.

Reply:
Yeah, so let's write `"The content's @type of a Message must be 'Mail', 'Request', 'ResponseWrapper', 'Notification' or 'ArbitraryMessageContent'."` or `"The content of a Message must either be a Mail, Request, ResponseWrapper, Notification or an ArbitraryMessageContent."`, right @jkoenig134? 😎 I'd prefer last, but first suggestion equally consistent to our formatting style and fine for me.

Reply:
I'm kind of ok with the second one, so let's take this.